README.md

```diff
@@ -187,6 +187,42 @@ values (like the API key) in source-controlled property files!
 
 ## Set Up Trust Between App and Conjur
 
+By default, the Conjur appliance generates and uses self-signed SSL certificates (Java-specific
```

Comment:
There is one minor issue here - self-signed certificates are not Java-specific - maybe we could remove description in parentheses, though I don't find this as an issue.